### ISSUE TITLE
Possible Inclusion: "Political competition, policy and growth: theory and evidence from the US"

### ISSUE THREAD

**Issue by svmiller** [joshuaalley/cross-sectional-ols#1]
Hi Joshua,

First of all, I love this repository. This is such a fantastic idea and a clear lacuna for those of us 1) teaching methods to grad students but 2) knowing of very few applied OLS examples, given the increasing complexity of models.

I wanted to alert you to a possible inclusion to this repository: Besley et al.'s (2010) ["Political Competition, Policy and Growth: Theory and Evidence from the US" ](https://academic.oup.com/restud/article-abstract/77/4/1329/1643021 )in the _Review of Economic Studies_.  I came across this article reading some instructional resources about simulating model parameters from the multivariate normal distribution. I was not able to find the formal replication data for this article (i.e. we weren't hand-wringing yet about replication) but I did find a critique of the article that appears to contain the data. [You can find that here](https://dataverse.harvard.edu/dataset.xhtml;jsessionid=a7ddbf238fb7f518164a3793c54f?persistentId=doi%3A10.7910%2FDVN%2FA87KGX&version=&q=&fileTypeGroupFacet=%22Text%22&fileAccess=).

Basically, it's a fairly simple OLS model on a relatively bite-sized data set. The only wrinkles are state and year fixed effects. I'm in the process of replicating Besley et al (2010) based on these data and already reproduced the first model in Table 2.

**Comment by svmiller** [comment]
I spoke too soon about the original data purportedly being unavailable: https://dataverse.harvard.edu/file.xhtml?persistentId=doi:10.7910/DVN/A87KGX/1YBWBY

**Comment by svmiller** [comment]
Here's a partial replication of Table 2 as I'm trying to figure out what columns coincide with what variables in the paper.

```r
library(tidyverse)
library(fixest)

PCPG <- haven::read_dta("~/Dropbox/projects/stevedata/data-raw/PCPG/formatteddata.dta")

# Model 1, Table 2
summary(feols(share_taxes_inc ~ compnorm, fixef = c("state", "year"), data = PCPG))
# Model 3, Table 2
summary(feols(share_taxes_inc ~ factor(state) + factor(year)  | compnorm ~ LT, data = PCPG))
# Model 4, Table 2
summary(feols(share_cap_exp ~ compnorm, fixef = c("state", "year"), data = PCPG))
# purportedly Model 8 of Table 2. I think it's actually Model 7
summary(feols(rtw ~ compnorm, fixef = c("state", "year"), data = PCPG))
```

**Comment by joshuaalley** [comment]
Hey Steven,

Thanks for passing this along, looks like a great fit. I'll get it loaded in sometime in the next couple of weeks. If you get a full replication file up and running, please drop it here or make a pull request. Otherwise I'll use the above code chunk for the replication script and add a note to the reference detailing which models the replication covers. 

I'm going to leave the issue open until I get it uploaded.

**Comment by joshuaalley** [comment]
On further examination, this is a good idea, but the panel data adds substantial complications. I therefore decided to make a sub-directory for panel models. Uploaded the data and a slightly tweaked script there. Thanks for passing this along!
